lib/ex_unit/lib/ex_unit/diff.ex

```diff
@@ -1162,6 +1162,7 @@ defmodule ExUnit.Diff do
     else
       other
       |> Map.to_list()
+      |> Enum.sort()
```

Comment:
@josevalim This is needed to make the ExUnit Diff Tests pass. Not sure if this is the best way...

Reply:
Yes, looks good to me. Feel free to submit it as a separate PR too. Your call.

Reply:
I'd like to write a failing test (also when running without coverage) and do a PR with the fix, but I'm not sure why this happens and I can only reproduce it when running it with coverage and the whole ex_unit suite.

Reply:
The order of maps is non-deterministic, it depends on which order modules are loaded, which you generally don't control, so you won't be able to reproduce it consistently (or at all).

Reply:
Ok, I will leave it in this PR then.